core/spec/lib/i18n_spec.rb

```diff
@@ -5,9 +5,15 @@
 
 RSpec.describe "i18n" do
   before do
+    # This reload avoids an issue with I18n.available_locales being cached
```

Comment:
Do you have ideas about why this is needed only in this specs and not in other files that uses `I18n`?

Reply:
I guess the other files don't need a new locale to be autodetected. We should keep an eye out for new failures related to this, though